extension/src/popup/components/manageAssets/ManageAssetRows/index.tsx

```diff
@@ -218,6 +246,39 @@ export const ManageAssetRows = ({
         </div>
         {children}
       </SimpleBar>
+      <LoadingBackground
+        onClick={() => {}}
+        isActive={blockedDomains.status === ShowOverlayStatus.IN_PROGRESS}
+      />
```

Comment:
you won't need to invoke this LoadingBackground with `SlideupModal` either 😃

Reply:
ah I forgot about SlideupModal 🤔  we can, it's a little different because I'm having the warning view  appear and sliding up at the same time. It looks like some refactoring to be able to use the SlideupModal, since it assumes it's always there below but slides up when needed (just tried seeing if it'd be a quick plug and play and I'm having some css trouble with it). If alright with you I'd also like to chalk it up as tech debt, we could change LedgerSign to use as well

Reply:
https://stellarorg.atlassian.net/browse/WAL-478

Reply:
Yup, sounds good. If it's not a quick 1:1 swap, we can hold off. Thanks for making the ticket!